### PR TITLE
fix: make general improvements

### PR DESCRIPTION
**Pull request by parishahla**

### CODE REVIEW

**Review by eyeamnoob — COMMENTED** [review]
Looks good so far.

File: src/utils/get-mail-plan-names.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export const getMailPlanName = (plan: any) => {
+  if (plan === 'm1') return 'Amber';
+  if (plan === 'm2') return 'Opal';
+  if (plan === 'm3') return 'Jadeite';
+  if (plan === 'm4') return 'Marble';
+  if (plan === 'm5') return 'Turquoise';
+  if (plan === 'm6') return 'Emerald';
+  if (plan === 'm7') return 'Ruby';
+  if (plan === 'm8') return 'Diamond';
+};
```
Comment (vashian) [inline src/utils/get-mail-plan-names.ts:10]: Mapping object to make it more concise and easier to maintain:

```js
export const getMailPlanName = (plan: string): string | undefined => {
  const planNames: Record<string, string> = {
    m1: 'Amber',
    m2: 'Opal',
    m3: 'Jadeite',
    m4: 'Marble',
    m5: 'Turquoise',
    m6: 'Emerald',
    m7: 'Ruby',
    m8: 'Diamond',
  };

  return planNames[plan];
};

```

File: src/base.ts [hunk 2]
```diff
@@ -246,12 +259,12 @@ Please check your network connection.`);
...
     if (!config['api-token'] || !config.region) {
-      const { api_token, region } = config.account
+      const { api_token } = config.account
         ? await this.getAccount(config.account)
         : await this.getCurrentAccount();
 
       config['api-token'] = config['api-token'] || api_token;
-      config.region = config.region || region;
+      // config.region = config.region || region;
     }
```
Comment (vashian) [inline src/base.ts:268]: Don't forget that all accounts have a `region` field in the global configuration file `liara-auth.json`

I believe we shouldn't modify this global config file. To eliminate the prompt asking for the region, we can simply set a fallback region to `"IRAN"` and never prompt the user for a region.

Additionally, we should only remove the `--region` flag for simplicity.

For backward compatibility, in my opinion, we should automatically set the region to "IRAN". This ensures nothing unexpected happens in the CLI and keeps the region field hidden from the user.

This approach will streamline the process and improve the user experience.

PING @mhemrg

Comment (parishahla) [reply src/base.ts]: > I believe we shouldn't modify this global config file. To eliminate the prompt asking for the region, we can simply set a fallback region to `"IRAN"` and never prompt the user for a region.

Fallback region has already been implemented. Please search for `FALLBACK_REGION` throughout the repo.

**Review by vashian — APPROVED** [review]

**Pull request merged**